LearningJournal/tox.ini

```diff
@@ -0,0 +1,8 @@
+[tox]
+envlist = py27, py35
+
+[testenv]
+commands = py.test /Users/admin-1/david_journal/LearningJournal/LearningJournal/LearningJournal/tests
```

Comment:
This file should enable _anyone_ to run tox, not just people who have the same directory structure as your machine. This path ought to be changed so that it's relative to the `tox.ini` file: something like `LearningJournal/tests`.